funnel/views/otp.py

```diff
@@ -272,6 +273,30 @@ def send_sms(
             return msg
         return None
 
+    # Send whatsapp message
+    def send_whatsapp(
+        self, flash_success: bool = True, flash_failure: bool = True
+    ) -> whatsapp.WhatsappTemplate | None:
```

Comment:
This needs a wrapping `send_phone` method that picks SMS or WhatsApp automatically — or both.